tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -30,7 +30,8 @@ const createNewFolderButton = '#new-folder-btn'
 const createNewTxtFileButton = '.new-file-btn-txt'
 const createNewMdFileButton = '.new-file-btn-md'
 const createNewDrawioFileButton = '.new-file-btn-drawio'
-const saveTextFileInEditorButton = '#text-editor-controls-save'
+const createNewOpenDocumentFileButton = '//ul[@id="create-list"]//span[text()="%s"]'
```

Comment:
it can be also used to create ms odocuments
```suggestion
const createNew(External|Document|Something else)FileButton = '//ul[@id="create-list"]//span[text()="%s"]'
```